app/rubin-2025/register-form.jsx

```diff
@@ -79,233 +79,294 @@ class RegisterForm extends React.Component {
         
-        <label className="form-separator">
-          <input
-            type="checkbox"
-            checked={this.state.underAge}
-            disabled={inputDisabled}
-            onChange={this.updateAge}
-          />
-          <Translate component="span" content="registerForm.underAge" />
-        </label>
-
-        <label>
-          <span className="columns-container inline spread">
-            <Translate content="registerForm.userName" />
-            {badNameChars?.length > 0 && (
-              <Translate className="form-help error" content="registerForm.badChars" />
-            )}
+        <div className="form-row">
+          <span className="checkbox-block">
+            <input
+              checked={this.state.underAge}
+              disabled={inputDisabled}
+              id="register-form-underAge"
+              name="underAge"
+              onChange={this.updateAge}
+              type="checkbox"
+            />
+            <label htmlFor="register-form-underAge">
+              <Translate component="span" content="registerForm.underAge" />
+            </label>
+          </span>
+        </div>
+
+        <div className="form-row">
+          <div className="label-block">
+            <label htmlFor="register-form-login">
+              <Translate content="registerForm.userName" />
+            </label>
+
+            <div aria-live="polite" id="register-form-login-help-message-1">
+              {badNameChars?.length > 0 && (
+                <Translate className="form-help error" content="registerForm.badChars" />
+              )}
+            </div>
+
             {"nameConflict" in this.state.pending && (
               <LoadingIndicator />
             )}
-            {nameConflict && (
-              <span className="form-help error">
-                <Translate content="registerForm.nameConflict" />{' '}
-                <a href={`${window.location.origin}/reset-password`}>
-                  <Translate content="registerForm.forgotPassword" />
-                </a>
-              </span>
-            )}
+
+            <div aria-live="polite" id="register-form-login-help-message-2">
+              {nameConflict && (
+                <span className="form-help error">
+                  <Translate content="registerForm.nameConflict" />{' '}
+                  <a href={`${window.location.origin}/reset-password`}>
+                    <Translate content="registerForm.forgotPassword" />
+                  </a>
+                </span>
+              )}
+            </div>
+
             {(this.state.input_login?.length > 0
               && nameConflict === false
+              && !("nameConflict" in this.state.pending)
             ) && (
               <span className="form-help success">
                 <Translate content="registerForm.looksGood" />
               </span>
             )}
+
             <Translate className="form-help info right-align" content="registerForm.required" />
```

Comment:
Add this to the form's accessible label so that screenreaders announce the field as required.

Reply:
Ditto for all the other required fields on the page.